### ISSUE TITLE
MultiSelect accessibility improvements

### ISSUE THREAD

**Issue by Juveniel** [telerik/kendo-ui-core#4642]
### General:

1. Aria-haspopup attribute is missing.

1. Consider adding aria-autocomplete="list" for announcing "listbox auto-completion list" when focusing the widget. ОК

1. Deleting an item is not announced.

### CrhomeVox Testing:

Selected options _not_ announced when the widget is focused. 

### NVDA Testing:

~~Navigating between selected items does not read them.~~

**Comment by jivanova** [comment]
You could use the following resources for reference.

https://www.w3.org/TR/wai-aria-practices-1.1/#combobox

https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

In order to announce changes in the multiselect, you might need to set `aria-live` attribute. Consider the below issues:

- [ ] https://github.com/telerik/kendo-ui-core/issues/4812
- [ ] https://github.com/telerik/kendo-ui-core/issues/2866

**Comment by Juveniel** [comment]
Based on the above reports, the following improvements to the MultiSelect will be added with the 2019 R2 release:

### Search Input
- [ ]  aria-haspopup
- [ ]  aria-autocomplete
- [ ]  aria-live

### Tag List
- [ ] role option to the selected items.
- [ ] aria-setsize to the selected items.

### Deleting an item
In order for the screen readers to announce the selected options after deleting an item from the tag list, the option must be focused with the keyboard arrows first and then deleted by pressing the `Backspace` or `Delete` keys.

In addition to the above, an accessibility article will be added to the Kendo UI MultiSelect documentation that describes the above in greater details.
